synapse/cortex.py

```diff
@@ -904,8 +891,38 @@ async def finidmon():
 
         await self.postNexsAnit()
 
-        if mirror is not None:
-            await self.initCoreMirror(mirror)
+        if self.mirror is not None:
+            await self._initCoreMirror(self.mirror)
+
+        # Enable leadership change awareness and fire
+        # the leadership hook once at boot
+        self.isleader = self.nexsroot.amLeader()
+        await self._initStormSvcs()
```

Comment:
stormservices have to be started up *after* leadership is established in the event we've got a serviceadd event that has to run.  This is possible if a stormservice def has been stamped into a cortex via hiveboot